samples/boards/stm32/mco/README.rst

```diff
@@ -0,0 +1,29 @@
+.. _samples_boards_stm32_mco:
+
+STM32 MCO example
+#################
+
+Overview
+********
+
+This sample is a minimum application to demonstrate how to output one of the internal clocks for
+external use by the application.
```

Comment:
Nit: align with other Zephyr samples (+ rewording)
```suggestion
This sample demonstrates how to use the Microcontroller Clock Output (MCO) capability of STM32 microcontrollers.

MCO can be used to output an internally generated clock signal on a GPIO pin of the MCU, for usage by external peripherals.
```

Reply:
The sample is already merged. Not part of this review.